docs/src/process.rst

```diff
@@ -172,6 +174,21 @@ Public members
         This is not supported on Windows, :c:func:`uv_spawn` will fail and set the error
         to ``UV_ENOTSUP``.
 
+.. c:member:: uv_process_options_t.cpumask
+.. c:member:: uv_process_options_t.cpumask_size
+
+    Libuv can set the child process' CPU affinity mask.  This happens when
+    `cpumask` is non-NULL.  It must point at an array of char values
+    of length `cpumask_size`, whose value must be at least that returned by
+    :c:func:`uv_cpumask_size`.  Each byte in the mask can be either
+    zero (false) or non-zero (true) to indicate whether the corresponding
```

Comment:
Out curiosity: any reason not to do this bitwise instead of byte wise?

Reply:
The API was worked out by discussion in #1389 and based on previous discussion in another MR for thread affinity.  IIUC the main reason for bytewise masking is simplicity.